woocommerce/woocommerce-gutenberg-products-block
# Handle Local Pickup Selection in the Cart

This is the final change to the cart page to sync shipping methods between packages when using Pickup Locations for multiple packages. 

Fixes #7897

### Screenshots

This is the view seen if you select Local Pickup while there are multiple packages in the cart:

![Screenshot 2022-12-15 at 15 17 07](https://user-images.githubusercontent.com/90977/207914085-694acb42-a068-44b2-b4ac-6ee527136865.png)

Given the need for the notice styling, I created a new component and extended the style to cover other notice types:

![Screenshot 2022-12-15 at 15 14 28](https://user-images.githubusercontent.com/90977/207914240-48bd2be9-2678-4ba3-9aa3-492b23fedd6b.png)

For context, this was the design:

<details>
<img width="399" alt="screen_shot_2022-12-12_at_12 16 00_pm" src="https://user-images.githubusercontent.com/90977/207913954-fec193c6-b6d3-4d19-9e1a-2c8f3a15baae.png">
</details>

### Testing

With a plugin that enables multiple packages:

1. Add items to your cart to get multiple packages/rates
2. Go to the cart page
3. Select a Local Pickup method. **Confirm the method is selected across all packages.**
4. Confirm the notice shows above the rate selector.
5. Select a non-local pickup method. **Confirm the method is selected across all packages.**. The notice is hidden.
6. Select another non-local pickup method. Other packages should not sync.


### WooCommerce Visibility
